.github/pull_request_template.md

```diff
@@ -22,7 +23,55 @@ Summarize the solution and provide any necessary context needed to understand
 the code change.
 -->
 
+## Follow Up Work
+
+<!--
+Is there anything missing from the solution?
+What still needs to be done?
+-->
+
 ## Related Issues
 <!--
 Please link to any existing GitHub issues pertaining to this PR.
 -->
+
+## Review Guidelines (for Reviewer)
+<!--
+This is a flexible checklist for the reviewer to fill in.
+
+Developers:
+Add extra tasks to the review using list items.
+Skip review tasks using ~~strikethrough~~, or just delete them.
+If you want this pull request to have a specific reviewer, tag them in the list of reviewers.
+
+Reviewer:
+This checklist can help you do your review.
+Add or skip tasks as needed.
+-->
+
+**Does this pull request improve Zebra?**
+
+- [ ] Pull Request
+  - **Is the pull request complete: code, documentation, and tests?**
+  - The PR contains changes to related code (unrelated changes can be split into another PR)
+  - The PR does everything listed in the tickets that it closes, or the designs that it finishes
+  - Any follow-up tasks are listed in a GitHub issue
+
+- [ ] Code
+  - **Does the code implement the specifications or design documents?**
+  - Changes from the specifications or design documents are explained in comments
+  - The code is readable, and does not appear to have any bugs
```

Comment:
The second item seems a little redundant.

```suggestion
  - The code prioritizes readability.
```

Reply:
What about...
```suggestion
  - The code is readable and maintainable
```